testsuite/forge.py

```diff
@@ -965,6 +965,19 @@ def image_exists(shell: Shell, image_tag: str) -> bool:
     return result.exit_code == 0
 
 
+def sanitize_forge_namespace(forge_namespace: str) -> str:
```

Comment:
why not regex like `re.sub(...)` ?

Reply:
This handles truncation as well and ultimately the implementation doesnt really matter that much